src/Breadcrumb.js

```diff
@@ -12,12 +12,13 @@ class Breadcrumb extends React.Component {
     const classes = getClassSet(bsProps);
 
     return (
-      <ol
-        {...elementProps}
-        role="navigation"
-        aria-label="breadcrumbs"
-        className={classNames(className, classes)}
-      />
+      <nav aria-label="breadcrumbs">
```

Comment:
I think the aria-label should probably be configurable. I'd make it an explicit prop but set the default to "breadcrumbs" that way a user can change the language or wording as needed